test/test_adrv9002_p.py

```diff
@@ -136,6 +136,12 @@ def test_adrv9002_str_attr(
 def test_adrv9002_interface_gain_narrowband(
     test_attribute_multipe_values_with_depends, iio_uri, classname, attr, depends, val
 ):
+    from adi.adrv9002 import adrv9002
+
+    sdr = adrv9002(iio_uri)
+    if sdr.rx0_sample_rate > 1000000:
```

Comment:
We should further improve this. Even though this might work most of the time, for correctness, we test `sdr.rx0_sample_rate > 1000000` when running the test for RX1 and `sdr.rx1_sample_rate > 1000000` when running for RX2

Reply:
Oh, okay. Thanks for pointing that out. I'd just edit the conditional then.

Reply:
I've added the sdr.rx1_sample_rate property to be checked on the conditional. Please see if this will cater to both of the channels.